OpenRA.Mods.Common/UtilityCommands/ExtractLuaDocsCommand.cs

```diff
@@ -45,8 +45,8 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 				"(note, you must replace the spaces in the snippet below with a single tab for each level of indentation):");
 			Console.WriteLine("```\nRules:\n\tWorld:\n\t\tLuaScript:\n\t\t\tScripts: myscript.lua\n```");
 			Console.WriteLine();
-			Console.WriteLine("Map scripts can interact with the game engine in three ways:\n" +
-				"* Global tables provide functions for interacting with the global world state, or performing general helper tasks.\n" +
+			Console.WriteLine("Map scripts can interact with the game engine in three ways:\n");
+			Console.WriteLine("* Global tables provide functions for interacting with the global world state, or performing general helper tasks.\n" +
```

Comment:
This statement contains many inline newlines. Using `Console.WriteLine` in the middle of it to add an extra linebreak isn't great style, and makes the `\n` at the end of the `Console.WriteLine` above look like an unintentional oversight.

Please either just insert an additional `\n` in the line above, or breaks the whole thing up into multiple `Console.WriteLine`s, with empty calls where we want a blank linebreak (like we already do above).

Reply:
Updated.